core/domain/exp_jobs_one_off_test.py

```diff
@@ -2097,7 +2097,8 @@ class ImagesAuditJobTests(test_utils.GenericTestBase):
     ALBERT_EMAIL = 'albert@example.com'
     ALBERT_NAME = 'albert'
 
-    VALID_EXP_ID = 'exp_id0'
+    VALID_EXP_ID = 'exp_id'
```

Comment:
Better to call this VALID_EXP_ID_1 and VALID_EXP_ID_2. Currently the convention you've chosen is rather asymmetric...

Reply:
Done